Doc/library/dis.rst

```diff
@@ -1071,7 +1108,7 @@ instructions:
 
 .. data:: hasconst
 
-   Sequence of bytecodes that have a constant parameter.
+   Sequence of bytecodes that have a constant argument.
```

Comment:
I think "`Sequence of bytecodes that access a constant.`" would be more accurate (if it is a correct English). *argc* is an index in the list of constants (the `co_consts` attribute of the code object).

Currently, in all maintained versions, `hasconst` contains only `LOAD_CONST`.

Reply:
Done.  I only touched it because I was searching for "parameter", because nearly all the existing uses of the word "parameter" were wrong.  But you're right, this one was completely wrong from the beginning.  Will update the PR in a sec.